backend/convert_to_protobuf.go

```diff
@@ -210,6 +211,7 @@ func (t ConvertToProtobuf) SubscribeStreamRequest(req *SubscribeStreamRequest) *
 	return &pluginv2.SubscribeStreamRequest{
 		PluginContext: t.PluginContext(req.PluginContext),
 		Path:          req.Path,
+		Query:         req.Query,
```

Comment:
@FZambia do you think this should be called "Data" rather than query?  The API will not enforce that this needs to match a particular query syntax and in general is just a way to make things less stateful